tests/js/setup.ts

```diff
@@ -219,6 +219,14 @@ jest.mock('@sentry/react', function sentryReact() {
         end: jest.fn(),
       }),
     }),
+    logger: {
+      warn: jest.fn(),
+      error: jest.fn(),
+      fatal: jest.fn(),
+      info: jest.fn(),
+      debug: jest.fn(),
+      trace: jest.fn(),
+    },
```

Comment:
Why did we need this change?

Reply:
Yeah this is generally missing. Could also extract to a separate PR?